samples/notebooks/openqasm.ipynb

```diff
@@ -0,0 +1,329 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "ae56fce0",
+      "metadata": {},
+      "source": [
+        "# QDK Interop with OpenQASM"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "2b838c23",
+      "metadata": {},
+      "source": [
+        "The modern QDK provides interoperability with OpenQASM 3 programs built upon the core QDK compiler infrastructure.\n",
+        "\n",
+        "This core enables integration and local resource estimation without relying on external tools. Users are able to estimate resources for their OpenQASM programs locally (see the [resource estimation with OpenQASM sample notebook](../../estimation/estimation-openqasm.ipynb)), leveraging the QDK compiler's capabilities for analysis, transformation, code generation, and simulation. This also enables the generation of QIR from OpenQASM progams leveraging the [modern QDKs advanced code generation capabilities](https://devblogs.microsoft.com/qsharp/integrated-hybrid-support-in-the-azure-quantum-development-kit/).\n",
```

Comment:
Isn't `../../estimation` one level to high if you're already in `samples/notebooks`?